server/service/src/apis/oms_central.rs

```diff
@@ -0,0 +1,60 @@
+use reqwest::{Client, StatusCode, Url};
+
+use super::api_on_central::NameStoreJoinParams;
+
+// Non-sync related APIs on the OMS Central server
+pub struct OmsCentralApi {
```

Comment:
This is the consumer side - btw should this be versioned now, or we can add later?